bedrock/settings/static_media.py

```diff
@@ -1165,6 +1165,14 @@
         ),
         'output_filename': 'js/firefox_developer_firstrun-bundle.js',
     },
+    # will be needed when videos are ready (shortly after 6/2?)
```

Comment:
Hmm, but there are videos already in the /whatsnew template, for locales that don't have the flag `l10n_has_tag('spring_release')`, or am I misunderstanding?